### PR TITLE
[OSCD] T1069.001: Local Groups

### PR DESCRIPTION
**Pull request by omergunal**
https://github.com/Neo23x0/sigma/tree/master/rules/linux

Task12

### CODE REVIEW

**Comment by yugoslavskiy** [comment]
Hello @omergunal ! 

Thank you for your contribution, once again (: 

Could you please apply the recommendations to this rule, as was provided for the rest of your rules?

File: rules/linux/lnx_local_group.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+detection:
+    keywords:
+        - keys|contains:
+          - 'groups'
+          - '/etc/passwd'
+          - '/etc/group'
+          - 'getent group'
+        - command:
+          - 'id'
+          - 'id *'
```
Comment (yugoslavskiy) [inline rules/linux/lnx_local_group.yml:20]: could you please use [this](https://github.com/Neo23x0/sigma/pull/1044#discussion_r502049487) recommendation here as well?

File: rules/linux/lnx_local_group.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+title: Local Groups
+id: 0d68a143-a178-4aa6-b3f5-e441aceca810
+status: stable
+description: Detects group enumeration
+author: Ömer Günal, oscd.community
+date: 2020/10/06
+references:
+    - https://attack.mitre.org/techniques/T1069/001/
```
Comment (yugoslavskiy) [inline rules/linux/lnx_local_group.yml:8]: could you please use [this](https://github.com/Neo23x0/sigma/pull/1044#discussion_r502045513) recommendation here as well?

**Comment by yugoslavskiy** [comment]
Hello @omergunal! Could you please fix the rule?

**Comment by yugoslavskiy** [comment]
@omergunal @alejandroortuno guys, it seems that you both developed a Sigma rule for technique T1069.001. 

Here is another rule this rule: https://github.com/Neo23x0/sigma/pull/1128/ 
The rule that was developed by @alejandroortuno has been approved, and I think it doesn't make sense to proceed with this (https://github.com/Neo23x0/sigma/pull/1056) PR. 

@alejandroortuno could you please mention @omergunal (`Ömer Günal`) in the `author` field of your rule, so we could close this PR as redundant? 

@omergunal could you please take a look at @alejandroortuno's rules and adjust the rest of your rules using them as an example, to improve the rest of your rules?

Could you guys check if there are any other intersections in your work and agree on the next steps in case there are?

Thank you very much!

**Comment by alejandroortuno** [comment]
Sure thing @yugoslavskiy apologies for the confusion. I have push a new commit to add @omergunal as the leading author on both ones for Linux and Macos. At first sight, haven's seen any other overlapping rules between my contributions and Ömer 's one.

**Comment by yugoslavskiy** [comment]
Hello @alejandroortuno! Thank you very much! 
@omergunal you can see the rules here: https://github.com/Neo23x0/sigma/pull/1128/files
Your authorship has been added. 
Sorry for the inconvenience, it's quite hard to manage so many contributions with our resources.
You can take a look at the rules and maybe collect some ideas regarding the rest of your PRs, @omergunal !  (:

I am going to close this PR as it's already worked out in https://github.com/Neo23x0/sigma/pull/1128/files

**Pull request closed without merging**